### ISSUE TITLE
Impossible to enter binary data for a barcode

### ISSUE THREAD

**Issue by andrewshadura** [ligi/PassAndroid#368]
I tried to store a local RailPlus pass in PassAndroid, but it has an Aztec code with binary (non-UTF-8) data, which doesn’t copy properly through the clipboard, resulting in a different Aztec code which wouldn’t scan.

When I tried to create an Espass JSON and manually encode it in a way I hoped I’d make it work, it crashed PassAndroid when I opened it.

**Comment by ligi** [comment]
Do you have a stack-trace by any chance?

**Comment by andrewshadura** [comment]
Not at the moment, but I can try again and see if I have anything.

**Comment by andrewshadura** [comment]
I’m curious what happens if you try az Aztec code from this project: https://github.com/pbock/ticket-parser
I suppose it should be in the same format as our railways (ZSSK).

**Comment by andrewshadura** [comment]
Right, easier said than done, I’m unable to transform this photo to be able to scan from it 😄

**Comment by andrewshadura** [comment]
![ZSSK specimen](https://user-images.githubusercontent.com/309253/156991329-75483234-fa47-4a04-a5bd-821447db3ee1.png)
